app/views/includes/dashboard-scripts.html

```diff
@@ -0,0 +1,2 @@
+{% include "includes/datatables-scripts.html" %}
```

Comment:
I've changed this to include scripts.html which doesn't include `jquery.dataTables.min.js`